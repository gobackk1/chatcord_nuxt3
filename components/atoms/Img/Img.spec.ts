import { shallowMount } from '@vue/test-utils'
import AtomImg from './Img.vue'

describe('UIコンポーネントのテスト', () => {
  describe('Viewのテスト', () => {
    describe('回帰テスト', () => {
      it('スナップショットテスト', () => {
        const wrapper = shallowMount(AtomImg, {
          props: {
            src: '/sample.jpg',
          },
        })
        expect(wrapper.element).toMatchSnapshot()
      })
    })
  })
})
