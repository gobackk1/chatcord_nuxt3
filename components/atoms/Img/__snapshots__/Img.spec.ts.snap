// Vitest Snapshot v1

exports[`UIコンポーネントのテスト > Viewのテスト > 回帰テスト > スナップショットテスト 1`] = `
<img
  src="/sample.jpg"
/>
`;
